tools/cdk-build-tools/bin/cdk-test.ts

```diff
@@ -67,12 +73,24 @@ async function main() {
     testCommand.push(args.nodeunit);
     testCommand.push(...testFiles.map(f => f.path));
 
-    await shell(testCommand, timers);
+    await shell(testCommand, { timers });
   }
 
   // Run integration test if the package has integ test files
   if (await hasIntegTests()) {
-    await shell(['cdk-integ-assert'], timers);
+    await shell(['cdk-integ-assert'], { timers });
+  }
+
+  // Run compatibility check if not disabled (against the latest
+  // published version)
+  if (!args.quick) {
+    try {
+      await shell([args["jsii-diff"], 'npm:'], { timers });
```

Comment:
It looks odd to have `npm:` here...

Reply:
In a newer drop of jsii-diff we can change it to `npm://` if that makes you feel better. But that is not out yet.

Reply:
It's more the fact that it looks like there should be something afterwards... And there's nothing...